packages/lab/src/navigation-item/NavigationItem.tsx

```diff
@@ -21,7 +19,7 @@ export interface NavigationItemProps extends ComponentPropsWithoutRef<"div"> {
    */
   active?: boolean;
   /**
-   * Whether the navigation item has active children.
+   * Whether the parent navigation item has active nested items.
```

Comment:
TBH, i don't quite the doc here v.s. the prop `blurSelected`

Reply:
I'll review the wording